packages/offix-datastore/src/replication/GraphQLCrudQueryBuilder.ts

```diff
@@ -55,15 +57,19 @@ export class GraphQLCrudQueryBuilder implements GraphQLQueryBuilder {
                   limit
               }
             }`,
-        sync: gql`
-                query sync${modelName}($lastChanged: String!, $filter: ${modelName}Filter) {
-                  find${modelName}s(lastChanged: $lastChanged, filter: $filter) {
-                      items {
-                        ${graphQLFields}
-                      }
-                      lastChanged
+        sync: {
+            query: gql`
+            query sync${modelName}($lastChanged: String!, $filter: ${modelName}Filter) {
+              find${modelName}s(lastChanged: $lastChanged, filter: $filter) {
+                  items {
+                    ${graphQLFields}
                   }
-                }`,
+                  lastChanged
+              }
+            }`,
+            getData: (response) => (response.data.items),
```

Comment:
-1 to have this two methods. I guess we will get that from the mentioned class

Reply:
Yes. It's a TODO